CHANGELOG.md

```diff
@@ -1,11 +1,26 @@
-# 2016-09-27 Release 0.6.1
+# Changelog
+
+## 2017-01-12 - Release 0.7.0
+
+Last release with Puppet 3 support!
```

Comment:
I know this is <1.0.0 but we should preface this with `BREAKING` to be sure people see it.